packages/app/src/components/applicationStages/ApplicationStages.js

```diff
@@ -45,16 +45,6 @@ const ApplicationStages = () => {
     }
   }
 
-  function closeApplication(value) {
```

Comment:
It looks like this PR has all the changes from your other PR as well. Make sure you pull the latest develop branch and branch off of that when you start a new ticket. It makes it hard to review when PRs have changes for multiple tickets in them.

Reply:
Thank you Austin for clarification. I will make sure to do that in the future.

Reply:
Can you merge develop into this branch to see if it fixes this now?